pinc/unicode.inc

```diff
@@ -170,3 +193,43 @@ function guess_string_encoding($text)
     # Give up and return ISO-8859-1
     return 'ISO-8859-1';
 }
+
+function make_title($codepoints)
+{
+    // make title from array of base codepoint and possible combining marks
+    $title = IntlChar::charName($codepoints[0]);
+    $count = count($codepoints);
+    if($count > 1)
+    {
+        $title .= " with " . IntlChar::charName($codepoints[1]);
```

Comment:
Creating the titles like this won't work for non-English speakers. It might simply be easier translation-wise if we removed the with/and altogether and just a `+` symbol between all of the titles instead.

Reply:
Good plan, I hadn't thought of translation. That would make the code simpler also.